packages/core/src/fhir-to-cda/cda-templates/components/__tests__/xmls/encounters-section-single-entry.xml

```diff
@@ -0,0 +1,171 @@
+<templateId root="2.16.840.1.113883.10.20.22.2.22" extension="2015-08-01">
+</templateId>
+<code code="46240-8" codeSystem="2.16.840.1.113883.6.1" codeSystemName="LOINC" displayName="History of encounters">
+</code>
+<title>
+	ENCOUNTERS
+</title>
+<text>
+	<table ID="encounters">
+		<thead>
+			<tr>
+				<th>
+					Reason for Visit
+				</th>
+				<th>
+					Encounter Type
+				</th>
+				<th>
+					Attending Provider
+				</th>
+				<th>
+					Location
+				</th>
+				<th>
+					Date
+				</th>
+			</tr>
+		</thead>
+		<tbody>
+			<tr ID="encounters1">
+				<td>
+					Viral sinusitis (disorder), Chronic pain (finding)
+				</td>
+				<td>
+					OV EST PT LEV 4
+				</td>
+				<td>
+					MD Zoidberg, John A.
+				</td>
+				<td>
+					Planet Express Medical Office - 1111 Sample Street, Suite 987, Springfield, CA 12123
+				</td>
+				<td>
+					07/23/2012 4:45 PM
+				</td>
+			</tr>
+		</tbody>
+	</table>
+</text>
+<entry>
+	<encounter classCode="ENC" moodCode="EVN">
+		<templateId root="2.16.840.1.113883.10.20.22.4.49" extension="2015-08-01">
+		</templateId>
+		<id root="placeholder-ORG-OID" extension="${encounterId}">
```

Comment:
Oh! TIL Lodash's template accept JS-like templates!!! 😍 

Why the hell didn't they add it to their doc? How did you find that out? 🤓

Reply:
I was using these placesholders with `eval()`, so I gave it a try with this method, and it worked 😅

Reply:
I'll try to move the E2E to this pattern too, next time I touch it - will add a note to the tickets.